statescript/executor.go

```diff
@@ -165,6 +208,18 @@ func execute(name string, timeout time.Duration) int {
 		return retCode(err)
 	}
 
+	var bts []byte
+	if stderr != nil {
+		bts, err = ioutil.ReadAll(stderr)
+		if err != nil {
+			log.Error(err)
+		}
+	}
+
+	if len(bts) > 0 && scriptHasLogStorageLeft(name, len(bts)) {
+		log.Error(string(bts))
```

Comment:
I would prefix this with some info which script is producing the logs:
`log.Errorf("stderr collected while executing script: %s [%s]", name, string(bts))`